docs/reference/technology/connectivity/wifi.md

```diff
@@ -33,6 +33,6 @@ As with Ethernet, Wi-Fi networks are only used for carrying IP traffic in Mbed O
 For more information, please see the following sections:
 
 - [IP networking](ip-networking.html).
-- [Ethernet](ethernet-technology.html).
+- [Ethernet](ethernet-technology.html). <!---Can links be like this, or do they need ../reference/?--->
```

Comment:
If they're in the same book, they work this way. If the inconsistency really bothers anyone, we can change it.